src/components/inputs/money-input/money-input.js

```diff
@@ -2,13 +2,47 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import invariant from 'tiny-invariant';
 import has from 'lodash.has';
+import Select from 'react-select';
+import ClearIndicator from '../../internals/clear-indicator';
+import DropdownIndicator from '../../internals/dropdown-indicator';
 import isNumberish from '../../../utils/is-numberish';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
 import Contraints from '../../constraints';
 import styles from './money-input.mod.css';
 import currencies from './currencies.json';
-import Currency from './currency';
-import CurrencyDropdown from './currency-dropdown';
+import createSelectStyles from '../../internals/create-select-styles';
+import * as vars from '../../../../materials/custom-properties.json';
```

Comment:
Nitpick: feels like the import name is not really descriptive. Just fearing a bit that it "spreads". Maybe `materialProperties` would be better? Not sure. It's also not `cssVariables` or `cssProperties` I suppose.

Reply:
I was also struggling to find a good name.

Those are all the `customProperties` we provide in CSS. In CSS we write  `var(--color-red)`, and in JS we can now write `vars['--color-red']`. I liked the similarity. I had `customProperties['--color-red']` at first.

With this explanation, would you be okay with keeping it?